apps/web/src/server/views/app/layouts/app-three-column.layout.njk

```diff
@@ -14,7 +14,7 @@
     {% endif %}
   {% endblock %}
   <div class="govuk-grid-row">
-    <div class="govuk-grid-column-one-quarter pins-content--sticky">
+    <div data-test-id="pageLeft" class="govuk-grid-column-one-quarter pins-content--sticky">
```

Comment:
Should we clear these data test ids when doing a prod release mode? I just don't like them :D

Reply:
What gets deployed to test environment? Is it prod release?